### PR TITLE
GEN-1401 - styles(carDealership): updated due design changes

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

UI changes for car dealership feature

## Justify why they are needed

Got final design from William.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3502** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3502" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3502?utm_source=stack-comment).

File: apps/store/src/features/carDealership/ProductItemContractContainer.tsx [hunk 1]
```diff
@@ -54,12 +56,12 @@ export const ProductItemContractContainerCar = (props: Props) => {
...
 const FIXTURE_CONTRACT = {
-  cost: { amount: 299, currencyCode: CurrencyCode.Sek },
+  cost: { amount: 579, currencyCode: CurrencyCode.Sek, reducedAmount: 299 },
 
-  endDate: '2024-12-01',
+  endDate: '2023-11-11',
 
   exposure: {
-    displayNameShort: 'ABC 123',
+    displayNameShort: 'LPP 083 ∙ Helförsäkring',
```
Comment (alebedev) [inline apps/store/src/features/carDealership/ProductItemContractContainer.tsx:64]: This should be displayNameFull, short is just reg number

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/features/carDealership/ProductItemContractContainer.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/features/carDealership/ProductItemContractContainer.tsx:64]: I see. Yes, it probably should. But we're in terms of changing that from moving away of fixtures and use API data.
Maybe we change that there?!

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Oct 31, 7:06 AM**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3502).

**Pull request merged**